lib/uring/uring.mli

```diff
@@ -91,42 +97,148 @@ module type FLAGS = sig
 
   val mem : t -> t -> bool
   (** [mem x flags] is [true] iff [x] is a subset of [flags]. *)
+
+  val empty : t
+  (** The empty set of flags *)
```

Comment:
Interface change as well: I think it should always be possible to pass in empty flags to the various calls here.